server/api/server.go

```diff
@@ -60,13 +59,18 @@ func (s *Server) Start(cfg config.Config) error {
 		s.registerMetrics()
 	}
 
+	addr := cfg.API.Address
+	if !strings.Contains(addr, "tcp://") {
```

Comment:
We might need better validation here, what if the address has `http://`

Reply:
Tendermint only allows `tcp://` and `unix://`, so if the address contains `http://` it will fail naturally.

Reply:
But I should clean this up to instead check if it has no schema at all.